game/state/city/vehicle.h

```diff
@@ -204,6 +204,7 @@ class Vehicle : public StateObject,
 	unsigned int shieldRecharge = 0;
 	int stunTicksRemaining = 0;
 	bool crashed = false;
+	bool evacuation = false;
```

Comment:
Does this need to be serialized too?

Reply:
Yes, really. It needs. Thanks for the tip.